bn_mp_radix_size.c

```diff
@@ -23,42 +21,52 @@ mp_err mp_radix_size(const mp_int *a, int radix, int *size)
       return MP_OKAY;
    }
 
-   /* special case for binary */
-   if (radix == 2) {
-      *size = mp_count_bits(a) + ((a->sign == MP_NEG) ? 1 : 0) + 1;
+   /* A small shortcut for powers of two. */
+   if (!(radix&(radix-1))) {
```

Comment:
how about making a private function out of this?

I'd say we don't even have put it in a separate file, simply here on the top and the prototype to `tommath_private.h`!?

Reply:
Yes I also thought about that.

Reply:
> how about making a private function out of this?

As said above, I would like to deprecate `mp_radix_size` in favor of "`mp_ilogb` plus a note in the documentation".
But I can do it if you insist, no prob'!

Reply:
okay, then we can also do the deprecation "now"!?

Reply:
let's discuss this above, there's more context already.